src/VisualStudio/Setup.ServiceHub/arm64/Roslyn.VisualStudio.ServiceHub.Setup.arm64.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepositoryEngineeringDir)targets\Services.props" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>net472</TargetFramework>
+    <IsShippingAssembly>false</IsShippingAssembly>
```

Comment:
>IsShippingAssembly [](http://example.com/codeflow?start=5&length=18)

The content of this file looks very similar to x64. Can we move all these common props and items to a shared props file and then just set the SetupProductArch?

Reply:
Done

Reply:
[From CI log](https://dev.azure.com/dnceng-public/public/_build/results?buildId=266223&view=logs&jobId=0f289b55-e51e-537b-a78d-659668d39b05&j=0f289b55-e51e-537b-a78d-659668d39b05&t=2704ecf5-7d4f-5fcd-84c9-45d42ed5f92e), seems BuildBoss needs TargetFramework property in the proj file
```
Processing Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.arm64.csproj
Project must new .NET SDK based

Processing Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.x64.csproj
Project must new .NET SDK based

Processing Roslyn.VisualStudio.ServiceHub.Setup.arm64.csproj
Project must new .NET SDK based

Processing Roslyn.VisualStudio.ServiceHub.Setup.x64.csproj
Project must new .NET SDK based

```